libs/community/langchain_community/chat_models/vertexai.py

```diff
@@ -214,6 +214,8 @@ class ChatVertexAI(_VertexAICommon, BaseChatModel):
     model_name: str = "chat-bison"
     "Underlying model name."
     examples: Optional[List[BaseMessage]] = None
+    raise_on_blocked: bool = False
```

Comment:
I don't think we need to change this, it would be deprecated soon.

Reply:
This an option I found in the VertexAI Python SDK, but I'm not completely sure if this could last long. I was inspired by the "OpenAIModerationChain" error argument.